Ejercicios/02 - Página Básica en ES6/src/ayudas.js

```diff
@@ -0,0 +1,5 @@
+import {colorAleatorio} from 'index.js';
+import {aleatorio} from 'index.js';
+
+
+export {colorAleatorio, aleatorio};
```

Comment:
```suggestion
export function colorAleatorio() {
  const r = aleatorio();
  const g = aleatorio();
  const b = aleatorio();
  return `rgb(${r}, ${g}, ${b})`;
}

export function aleatorio() {
  return Math.floor(Math.random() * 255);
}
```

La idea es que en este archivo estén las funciones que consideramos ayudas generales para cualquier parte de la aplicación y exportar cada una.